File: smartrefreshlibrary/src/main/java/com/zlin/smartrefresh/utils/DrawableUtils.java
package com.zlin.smartrefresh.utils;

import com.zlin.smartrefresh.config.BallInfoConfig;

public class DrawableUtils {

    public static float[] getDrawableSize(BallInfoConfig ballInfoConfig){
        float mBallRadius=ballInfoConfig.getBallRadius();
        float mBallHgap=ballInfoConfig.getBallHgap();
        float mBallVgap=ballInfoConfig.getBallVgap();

        float mDrawableWidth=3*(2*mBallRadius)+2*mBallHgap;
        float mDrawableHeight=2*mBallRadius+mBallVgap;

        return new float[]{mDrawableWidth, mDrawableHeight};
    }

}

File: smartrefreshlibrary/src/main/java/com/zlin/smartrefresh/utils/DrawableUtils.kt
package com.zlin.smartrefresh.utils

import com.zlin.smartrefresh.config.BallInfoConfig

object DrawableUtils {

    @JvmStatic
    fun getDrawableSize(ballInfoConfig: BallInfoConfig): FloatArray {
        val mBallRadius = ballInfoConfig.ballRadius
        val mBallHgap = ballInfoConfig.ballHgap
        val mBallVgap = ballInfoConfig.ballVgap
        val mDrawableWidth = 3 * (2 * mBallRadius) + 2 * mBallHgap
        val mDrawableHeight = 2 * mBallRadius + mBallVgap
        return floatArrayOf(mDrawableWidth, mDrawableHeight)
    }

}